examples/swift/quickstart/Package.swift

```diff
@@ -0,0 +1,20 @@
+import PackageDescription
+
+let package = Package(
+    name: "QuickstartExample",
+    platforms: [
+        .macOS(.v12)
+    ],
+    dependencies: [
+        .package(url: "https://github.com/bitcoindevkit/bdk-swift", from: "0.30.0")
```

Comment:
it looks like its pulling in pre-1.0, I'm not sure if that's on purpose or not (or if you want to use the most recent 1.0 tag of beta 5)

Reply:
hahah no no we would need to update it. I honestly wasn't paying much attention and let cursorAI do most the work. the real question is whether you think it makes sense to have lightweight script projects for swift and kotlin, or if we need to assume the reader is in xcode/androidstudio. currently it attempts to cover both, but what I'm proposing is that its good to have actual runnable scripts for the code in every language so we can ensure the code works and is up to date

Reply:
ya the kotlin version is also pulling in an old version of bdk. I wasn't thinking about that at the time. certainly would need to be updated, but the question here is just about direction/organization of the book

Reply:
I do love the addition of the code being consolidated https://github.com/bitcoindevkit/book-of-bdk/blob/master/examples/swift/quickstart/main.swift (and potentially runnable in CI) so kudos on that! I don’t have a strong opinion off the top of my head for `the real question is whether you think it makes sense to have lightweight script projects for swift and kotlin, or if we need to assume the reader is in xcode/androidstudio.`,  my main thoughts previously were just contained to my code and next steps I laid out in https://github.com/bitcoindevkit/book-of-bdk/pull/42 but I’m happy to see some of the wider scoping thoughts and I'll keep thinking about this.  And then I can review the swift code once it’s updated to the 1.0

Reply:
ok cool, so maybe this approach does make more sense for swift than kotlin. I'm happy to update the swift code to the latest bdk version. @thunderbiscuit said he wanted to work out the kotlin example himself since he is most experienced with android dev, but it sounds like you don't feel as strongly about the swift example so I'll go ahead and do what I can on that